submit/templates/submit/withdraw.html

```diff
@@ -0,0 +1,79 @@
+{%- extends "base/base.html" %}
+
+{% import "submit/submit_macros.html" as submit_macros %}
+
+{% block addl_head %}
+ <link rel="stylesheet" href="{{ url_for('static', filename='css/submit.css')}}" />
+ <script src="{{ url_for('static', filename='js/filewidget.js') }}"></script>
+{% endblock addl_head %}
+
+{% block content %}
+  <h1 class="title">Update journal reference</h1>
+  <p>
```

Comment:
We may want some presentation markup - another box, or some other layout. Easy to skip over as a plain full-width paragraph.